### PR TITLE
GIT22310: Added a note in step 3 about quay.io persisting in the regstry after mirroring.

### LINKED ISSUE #22310: [enterprise-4.4] Mirrored images still show quay.io as registry. Make a hint for that.
Hi,

today we mirrored the OKD 4 images for Beta4 to our internal Artifactory registry (but this will also be an issue for OCP).

We were confused that podman images still shows quay.io as the registry on the bootstrap VM.

@vrutkovs explained to me that this is working as expected, because the image name gets "patched" to quay.io during the mirroring. We searched for two hours for the reason of the problem that was no problem at all :-)

Create a note about that in the official docs, please. That might help others who are stumbling across the same problem.

https://docs.openshift.com/container-platform/4.4/installing/install_config/installing-restricted-networks-preparations.html#installation-mirror-repository_installing-restricted-networks-preparations

Thanks and greetings,

Josef

### PR DESCRIPTION
**Pull request by jonquilwilliams**
Fixes #22310 

QE needed.

For peer reviewer: Please merge and CP to 4.3, 4.4, 4.5, 4.6, 4.7 after it passes QE review. Thanks.

### CODE REVIEW

File: modules/installation-mirror-repository.adoc [hunk 1]
```diff
@@ -145,6 +145,10 @@ the `imageContentSources` data that you require when you install your cluster.
 
 ... Record the entire `imageContentSources` section from the output of the previous
 command. The information about your mirrors is unique to your mirrored repository, and you must add the `imageContentSources` section to the `install-config.yaml` file during installation.
+[NOTE]
```
Comment (bmcelvee) [inline modules/installation-mirror-repository.adoc:148]: This is building a bit wonky. Try adding a blank line before the [NOTE].

Comment (bmcelvee) [reply modules/installation-mirror-repository.adoc:148]: s/quay.io/Quay.io

**Comment by bmcelvee** [comment]
@jonquilwilliams just a few comments, otherwise LGTM!

I don't think this requires QE review since it's a minor note, and not adding extensive or procedural content. But I'll defer to @vikram-redhat  to make a final call on that.

File: modules/installation-mirror-repository.adoc (hunk 1 above)
Comment (jonquilwilliams) [reply modules/installation-mirror-repository.adoc:150]: Your edits have been incorporated. Thank you!

**Comment by vikram-redhat** [comment]
Yes, let's QE this change.

@xltian - when you are back, could you assign the right associate to validate this?

**Comment by gpei** [comment]
Looks good for QE, thanks.

**Pull request merged**